botcore/utils/loggers.py

```diff
@@ -0,0 +1,45 @@
+"""Custom :obj:`logging.Logger` class that implements a new ``"TRACE"`` level."""
```

Comment:
Maybe we should make this module more generic in case we want to add other logging utilities in the future. Thus, `logging.py` would be a better name (if it doesn't cause issues conflicting with stdlib's name) or something like `log.py`.

Reply:
if we would use logging, then we'd need to namespace or alias it. I don't think log would work, since that's the name we give to the isntances of our logger in the bots themselves.

i think using logging is fine, since we shouldn't really have any file in the bot that imports both.

Reply:
https://github.com/python-discord/bot-core/pull/29/commits/221b4985ee88845b4d93487969d47d6b9edeba57